Integrations/integration-TruSTAR.yml

```diff
@@ -147,6 +168,136 @@ script:
+            })
+            if threshold <= indicator_score:
+                file_ec.update({
+                    'Malicious': {
+                        'Vendor': 'TruSTAR'
+                    }
+                })
+        return {
+            'DBotScore': dbot_ec,
+            'File': file_ec,
+            'TruSTAR.File': trustar_ec
+        }
+
+    def create_ip_ec(indicators, ip, threshold):
+        trustar_ec = {}
+        ip_ec = {}
+        dbot_ec = {}
+        for indicator in indicators:
+            ip_ec.update({
+                'Address': indicator['value'],
+            })
+            trustar_ec.update({
+                'Value': indicator['value'],
+                'Whitelisted': indicator['whitelisted'],
+                'Priority': indicator['priorityLevel']
+            })
+            indicator_score = priority_level_to_score(indicator["priorityLevel"])
+            dbot_ec.update({
+                'Indicator': ip,
+                'Type': 'ip',
+                'Vendor': 'TruSTAR',
+                'Score': 0 if indicator_score == 0 else (2 if threshold > indicator_score else 3)
+            })
+            if threshold <= indicator_score:
+                ip_ec.update({
+                    'Malicious': {
+                        'Vendor': 'TruSTAR',
+                        'Description': indicator['priorityLevel']
+                    }
+                })
+        return {
+            'DBotScore': dbot_ec,
+            'IP': ip_ec,
+            'TruSTAR.IP': trustar_ec
+        }
+
+    def create_url_ec(indicators, url, threshold):
+        trustar_ec = {}
+        url_ec = {}
+        dbot_ec = {}
+        for indicator in indicators:
+            url_ec.update({
+                'Data': indicator['value'],
+            })
+            trustar_ec.update({
+                'Value': indicator['value'],
+                'Whitelisted': indicator['whitelisted'],
+                'Priority': indicator['priorityLevel']
+            })
+            indicator_score = priority_level_to_score(indicator["priorityLevel"])
+            dbot_ec.update({
+                'Indicator': url,
+                'Type': 'url',
+                'Vendor': 'TruSTAR',
+                'Score': 0 if indicator_score == 0 else (2 if threshold > indicator_score else 3)
+            })
+            if threshold <= indicator_score:
+                url_ec.update({
+                    'Malicious': {
+                        'Vendor': 'TruSTAR',
+                        'Description': indicator['priorityLevel']
+                    }
+                })
+        return {
+            'DBotScore': dbot_ec,
+            'URL': url_ec,
+            'TruSTAR.URL': trustar_ec
+        }
+
+    def create_domain_ec(indicators, url, threshold):
+        trustar_ec = {}
+        domain_ec = {}
+        dbot_ec = {}
+        for indicator in indicators:
+            domain_ec.update({
+                'Name': indicator['value'],
```

Comment:
Is there no data which can do under `Domain` other than `Name` ?

Reply:
@mayagoldb ?